test/e2e/e2e_visibility_on_demand_test.go

```diff
@@ -140,6 +146,219 @@ var _ = Describe("VisibilityOnDemand", Label("visibility-on-demand"), Ordered, f
 			}, testutils.OperatorReadyTime, testutils.OperatorPoll).Should(Equal(int32(2)), "Nominal concurrency shares did not change back to the default")
 		})
 	})
+
+	When("PendingWorkloads list should be checked for a ClusterQueue and LocalQueue", func() {
+		var (
+			labelKey   = "kueue.openshift.io/managed"
```

Comment:
The definition in the constants file could be used instead.
https://github.com/openshift/kueue-operator/blob/main/test/e2e/testutils/constants.go#L25